src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -1,27 +1,23 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Net;
-using System.Runtime.Serialization.Formatters.Binary;
-using System.Text;
-using System.Threading.Tasks;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using Orleans.Runtime;
 
 namespace Orleans.Serialization
 {
-    internal class OrleansJsonSerializer : IExternalSerializer
+    public class OrleansJsonSerializer : IExternalSerializer
     {
-        public static JsonSerializerSettings settings;
+        public static JsonSerializerSettings SerializerSettings { get; set; }
```

Comment:
I think I'm missing something here - why is it exposed as a static property on `OrleansJsonSerializer` and not as part of a config object? Is it desirable for the settings to be flipped back and forth at run time? I hope not. If this is just to configure the serializer once, then the settings should be immutable, right?